.github/workflows/go-test.yaml

```diff
@@ -0,0 +1,83 @@
+name: Go Test
+
+on:
+  pull_request:
+  push:
+
+jobs:
+  test:
+    name: Test
+    runs-on: ubuntu-latest
+    timeout-minutes: 60
+
+    steps:
+      - name: Check out code
+        uses: actions/checkout@v4
+
+      - name: Set up Go
+        uses: actions/setup-go@v4
+        with:
+          go-version: "1.24"
```

Comment:
You can use the version from go.mod to make it easier to maintain:

```yaml
go-version-file: go.mod
```

Same for other workflows.